components/nexus-repository/src/test/java/org/sonatype/nexus/repository/rest/internal/resources/SearchResourceTest.java

```diff
@@ -480,7 +480,7 @@ public void testGetAssetParams() {
     Set<String> allKeys = searchUtils.getSearchParameters().keySet();
     allKeys.forEach(s -> sb.append(s).append("=valueDoesNotMatter&"));
 
-    // asert only assert params remain
+    // assert only assert params remain
```

Comment:
Should this second "assert" be "asset"?

Reply:
I'm really not sure. That one definitely puzzled me a bit. Thinking about it again, you may be right.